Rakefile

```diff
@@ -317,6 +325,7 @@ namespace :spec do
              :rails, :railsredis, :railsredis_activesupport, :railsactivejob,
              :elasticsearch, :http, :redis, :sidekiq, :sinatra, :hanami, :hanami_autoinstrument]
 
+  desc '' # "Explicitly hiding from `rake -T`"
```

Comment:
Nope. I feel the pain as well.

Because `RSpec::Core::RakeTask` has its own logic.
https://github.com/rspec/rspec-core/blob/main/lib/rspec/core/rake_task.rb#L111

Reply:
yep, I tried it in datadog-ci-rb - did not work